src/sources/prometheus/mod.rs

```diff
@@ -1,13 +1,16 @@
 use crate::{
+    hyper::body_to_bytes,
     internal_events::{PrometheusHttpError, PrometheusParseError, PrometheusRequestCompleted},
     shutdown::ShutdownSignal,
-    stream::StreamExt,
+    stream::StreamExt as VStreamExt,
     topology::config::GlobalOptions,
     Event,
 };
-use futures01::{sync::mpsc, Future, Sink, Stream};
+use futures::{FutureExt, TryFutureExt};
+use futures01::{sync::mpsc, Future, Sink, Stream as Stream01};
 use http::Uri;
-use hyper_openssl::HttpsConnector;
+use hyper13::{Body, Client, Request};
+use hyper_openssl08::HttpsConnector;
```

Comment:
Gosh it would be so nice to remove some of these versioned deps. :(

Reply:
They will, this and one more PR, and they are out of here. At least those.